src/test/java/core/basesyntax/service/impl/TransactionServiceImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.impl.FruitStorageDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.strategy.TransactionHandler;
+import core.basesyntax.strategy.TransactionHandlerStrategy;
+import core.basesyntax.strategy.impl.BalanceTransactionHandler;
+import core.basesyntax.strategy.impl.PurchaseTransactionHandler;
+import core.basesyntax.strategy.impl.ReturnTransactionHandler;
+import core.basesyntax.strategy.impl.SupplyTransactionHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionServiceImplTest {
+    private static FruitStorageDao dao;
+    private static Map<FruitTransaction.Operation, TransactionHandler> map;
+    private static TransactionHandlerStrategy strategy;
+    private static TransactionService transactionService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        dao = new FruitStorageDaoImpl();
+        map = new HashMap<>();
```

Comment:
why do you save it in fields?

Reply:
and strategy too

Reply:
Because I foloved a sample which I saw durin a lesson. This video https://vimeo.com/showcase/9696169/video/745337080

Reply:
Do I have to change it?